src/bionics.cpp

```diff
@@ -430,7 +432,16 @@ bool Character::activate_bionic( int b, bool eff_only )
             }
         }
 
-        const size_t win_h = std::min( static_cast<size_t>( TERMY ), bad.size() + good.size() + 2 );
+        const auto &vit_all = vitamin::all();
```

Comment:
Please replace auto with the proper type for non-iterator and non-lamba types (such as this).